analyses/cms-open-data-ttbar/statistical.py

```diff
@@ -0,0 +1,471 @@
+import ROOT
+import os
+from statistical_interference.agc_sample import AGC_Sample
+from statistical_interference.rebinning_tool import RebinningTool
+from statistical_interference.drawer import Visualization, DrawModel
+
+
+def fit_histograms(filename=""):
+    print(f"Starting fitting process using input file: {filename}")
+
+    os.makedirs("statistical_data", exist_ok=True)
+
+    rebinning_tool = RebinningTool()  # Rebinning as extra step
+    rebinning_tool.set_xmin(110)
+    rebinning_tool.set_rebin(2)
+    rebinning_tool.set_input_path(filename)
+    rebinning_tool.set_output_path("statistical_data/temp_histos.root")
+    rebinning_tool.apply_rebinning()
+
+    file = ROOT.TFile(
+        "statistical_data/HistFactoryExtra.root", "RECREATE"
+    )  # cleaning the file
+    file.Close()  # create separate file for each file -> two much files
+
+    input_file = "statistical_data/temp_histos.root"  # input file for futher defined histograms
+
+    meas = ROOT.RooStats.HistFactory.Measurement("meas", "meas")
+    meas.SetLumi(1.0)
+    meas.SetLumiRelErr(0.0)
+
+    # ! main difference
+    lumi_systematics = (
+        ROOT.RooStats.HistFactory.OverallSys()
+    )  # we define it one time -> dont need to redefine it
+    lumi_systematics.SetName("Lumi")
+    lumi_systematics.SetLow(0.97)
+    lumi_systematics.SetHigh(1.03)
+
+    channel = ROOT.RooStats.HistFactory.Channel("channel_4j1b_CR")
+    channel.SetData("4j1b_pseudodata", input_file)
+    channel.SetStatErrorConfig(0.001, "Gaussian")
+
+    ttbar = AGC_Sample("ttbar", "4j1b_ttbar", input_file)
+    ttbar.SetSystematicsInputFile(
+        input_file
+    )  # if use SetInputFile -> change input file for nominal histogram
+    # we can put input file from sample data definition as default
+
+    ttbar.AddOverallSys(lumi_systematics)
+    ttbar.ActivateStatError()  # ROOT does not have DisableStatError() method, so we have to keep this :(
+    ttbar.AddNormPlusShapeHistoSys(
+        "ME_variation", histoname_up="4j1b_ttbar_ME_var"
+    )  # "histogram_up" can be skipped here
```

Comment:
```suggestion
    # "histogram_down" can be skipped here:
    ttbar.AddNormPlusShapeHistoSys("ME_variation", histoname_up="4j1b_ttbar_ME_var")  
```
You mean "histogram_down"? I don't see that histoname_up is skipped. Also, I would better place these comments in a separate line before, to avoid ugly line breaks in the code formatter.